CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 # Changelog
 
-## TBD
+## 5.5.0 (2021-01-07)
+
+* Store user information in persistenceDirectory
```

Comment:
Can you add a summary of the changes before the PR list to describe the net effect of the changes? i.e. clients can now be started in two processes independently and reference the new config options involved.

Reply:
(Happy for this to be done as part of release PR to avoid CI re-run.)